src/esssans/isis/visualization.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+"""
+Plotting functions for ISIS data.
+"""
+from typing import Any
+
+import scipp as sc
+
+
+def plot_instrument(da: sc.DataArray, pixels_per_tube: int = 512, **kwargs: Any) -> Any:
```

Comment:
Are you thinking this is generic enough that it can be used for other isis instruments? I guess it would work for sans2d as well. But then it should also work for other flat panels, not just isis panels?
Should we call it something else that implies it is plotting a flat panel? As it wouldn't work for something else.

Reply:
Are there any ISIS SANS instruments that do not have tubes that we have to worry about?

Reply:
I renamed it.